docs/HISTORY.txt

```diff
@@ -5,6 +5,7 @@ Changelog
 ------------------
 
 - Drop Plone 4.1 support. [jone]
+- Drop Plone 4.2 support. [busykoala]
```

Comment:
Please move this entry above `Drop Plone 4.1 support. [jone]`. The latest changes should be on the top of the file since we're reading from top (latest release) to bottom (oldest release).

Reply:
👍 I swapped the two lines now.